backend/plugins/frontline_api/src/modules/integrations/facebook/db/models/Bots.ts

```
@@ -0,0 +1,306 @@
+            text: greetText,
+          },
+        ];
+      }
+
+      await graphRequest.post('/me/messenger_profile', pageAccessToken, doc);
+
+      return { status: 'success' };
+    }
+
+    static async disconnectBotPageMessenger(_id) {
+      const bot = await this.getBot(_id);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)